docs/start/hello-world.ipynb

```diff
@@ -7,23 +7,64 @@
    "source": [
```

Comment:
<p>"You can also watch this Hello World in practice in the Hello World episode of the Coding with Qiskit 1.0 video series"</p><p> </p><p>I found this sentence difficult to parse, is there a typo?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/1158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>